src/pages/publication/NewPublication.tsx

```diff
@@ -53,6 +54,11 @@ const NewPublication: FC = () => {
   const [expanded, setExpanded] = useState<string | false>(false);
   const [showForm, setShowForm] = useState(false);
   const { t } = useTranslation();
+  const [uppy] = useState(createUppy());
```

Comment:
denne er veldig snål, men finner rett og slett ikke en bedre løsning...
- hvis vi ikke bruker state vil variabelen initieres på nytt per render -> ikke bra
- hvis vi flytter initiering til utenfor komponenten vil den ikke re-initialiseres for hver onMount (med mindre vi gjør det i en `useEffect`, men da blir det mer kode og en ekstra unødvendig render) -> ikke bra

Reply:
noe sier meg at `uppy` her er en veldig god kandidat til å legges i React context også, men vet ikke om det vil fjerne problemet heller

Reply:
stod det ikke at man ikke skulle initialisere Uppy i en FC?
https://uppy.io/docs/react/dashboard/#CSS